variables.tf

```
@@ -329,6 +330,7 @@ variable "node_pools" {
     node_count_min_per_zone    = 1
     node_count_max_per_zone    = 2
     node_labels                = {}
+    node_resource_labels       = {}
```

Move `node_resource_labels` up to keep `node_labels` and `node_taints` together as they are related to each other.
```suggestion
    node_resource_labels       = {}
    node_labels                = {}
```